setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.63.1',
+      version='0.64.1',
```

Comment:
```suggestion
      version='0.64.0',
```
Is there a reason you didn't reset patch value?

Reply:
not a good one